vscode/src/createProject.ts

```diff
@@ -265,93 +254,175 @@ export async function initProjectCreator(context: vscode.ExtensionContext) {
           return;
         }
 
-        // Find all the other Q# projects in the workspace
-        const projectFiles = (
-          await vscode.workspace.findFiles("**/qsharp.json")
-        ).filter((file) => file.toString() !== qsharpJsonUri.toString());
-
-        const projectChoices: Array<{ name: string; ref: Dependency }> = [];
-
-        projectFiles.forEach((file) => {
-          const dirName = file.path.slice(0, -"/qsharp.json".length);
-          const relPath = getRelativeDirPath(qsharpJsonDir.path, dirName);
-          projectChoices.push({
-            name: dirName.slice(dirName.lastIndexOf("/") + 1),
-            ref: {
-              path: relPath,
-            },
-          });
-        });
-
-        Object.keys(githubProjects).forEach((name) => {
-          projectChoices.push({
-            name: name,
-            ref: githubProjects[name],
-          });
-        });
-
-        // Convert any spaces, dashes, dots, tildes, or quotes in project names
-        // to underscores. (Leave more 'exotic' non-identifier patterns to the user to fix)
-        //
-        // Note: At some point we may want to detect/avoid duplicate names, e.g. if the user already
-        // references a project via 'foo', and they add a reference to a 'foo' on GitHub or in another dir.
-        projectChoices.forEach(
-          (val, idx, arr) =>
-            (arr[idx].name = val.name.replace(/[- "'.~]/g, "_")),
+        // ask the user if they want to import from github, or locally on their computer
+        const importChoice = await vscode.window.showQuickPick(
```

Comment:
I'm not sure we need to bifurate based on project location here and add an extra UI step/click for the user. In the vast majority of cases I expect users will have zero to maybe 1 or 2 local projects in their workspace, so the current behavior of listing local projects first followed by GitHub to choose from seems fine. (The 'local directory' wording could also be misleading, as we don't allow you to choose any local directory, only those projects that are already open in the current workspace).

Reply:
Alright, the only reason I did it this way is because they are different flows -- the github flow asks one more question than the local project flow (the `ref`), so conflating them made it a bit messy. But I'll merge them and keep it as clean as possible

Reply:
I though you said you addressed this? I still see the quick pick to choose between GitHub or local project. Or was there a misunderstanding?

Reply:
Actually. Don't worry about it. It's fine for now.